src/controllers/onboardingController.ts

```diff
@@ -148,9 +150,19 @@ export async function postForm(req, res) {
         'Autre'].includes(domain)) {
         return domain;
       }
-      formValidationErrors.push(`${field} : le domaine n'est pas valide`);
+      formValidationErrors[field] = 'Le domaine n‘est pas valide';
       return null;
     }
+    const isPublicServiceEmail = async function (field, email, emailBetaAsked) {
```

Comment:
la méthode mélange validation d'un formulaire et appelle réseau pour valider, ça devrtait être séparé.
Separation of concern